tests/scripts/build-ol.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+_base_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
+context_dir="${_base_dir}/image"
+
+curl -L "https://yum.oracle.com/templates/OracleLinux/OL8/u10/x86_64/OL8U10_x86_64-kvm-b258.qcow2" -o "${context_dir}/OL8U10-amd64.qcow2"
```

Comment:
Any objections to writing up an issue to introduce a basic bump script for these URIs if we are not able to ship them as containerdisks for whatever reason? It's an optional thing to implement if we end up having to manually bump this too often in the end.